src/templates/page.scss

```diff
@@ -528,4 +528,57 @@ code[class*="language-"] {
 .ais-SearchBox-input{
   outline: none!important;
   width: 80px;
+}
+.card_container{
+  padding-bottom: 2rem;
+}
+.chat_btn{
+  padding: 1rem;
+  margin-right: 1rem;
+  background: linear-gradient(0deg, #183E46, #183E46), #03A973;
+  border-radius: .2rem;
+  span{
+    padding-left: .5rem;
+    color: $white;
+    font-weight: 600;
+    font-size: 17px;
+  }
+}
+.community_btn{
+  padding: 1rem;
+  border: 1px solid #183E46;
+  border-radius: 2px;
+  span{
+    padding-left: .5rem;
+    color: #183E46;
+    font-weight: 600;
+    font-size: 17px;
+  }
+  @media (max-width: 425px) {
+    margin-top: 1rem;
+  }
+}
+.homepage_footer{
+  flex-basis: 100%;
+  justify-content: space-between;
+  margin-bottom: 4rem;
+  .pagination_buttons{
+    @media (min-width: 768px) {
+      margin: 0;
+    }
+  }
+  .next_button{
+    width: fit-content;
+    @media (min-width: 768px) {
+      margin-top: 1rem;
+    }
+  }
+}
+.home-page_banner{
+  background-image: url("~/src/components/images/bg-banner.svg");
```

Comment:
why it's not inside assets?

Reply:
The docs and tutorials post content images are rendered from the AWS cloud and only very few images are used in components, that's why those images are saved inside of components. Also, only the Docs team has access to AWS.